### DISCUSSION TITLE
cfclient takeoff and landing button in custom apps

### DISCUSSION THREAD

**Discussion by jonas-kupferschmid** [bitcraze/discussions#710] [Q&A]
Hello!

I wrote a custom app in the crazyflie firmware, however, I noticed that I am not anymore able to use the Command based Flight control buttons anymore. Especially the takeoff and landing buttons. How can I add them back in? and where can I see the default template of it being used (when no app is being run)?

Thank you in advance!

**Comment by knmcguire** [comment]
Technically the client checks if the Crazyflie has any of these attached decks: `position_decks = ['bcFlow', 'bcFlow2', 'bcLighthouse4', 'bcLoco', 'bcDWM1000']`

https://github.com/bitcraze/crazyflie-clients-python/blob/79247904d0da853d0e33913154cdc8c23dde9490/src/cfclient/ui/tabs/FlightTab.py#L356-L367

Are you using any of these decks for the crazyflie? And have you restarted the cfclient?
